### ISSUE TITLE
You should not be able to join an ongoing game

### ISSUE THREAD

**Issue by RedDragon98** [SpicyDragon95/xplex-issues#11]
in spleef

**Comment by SnowHuskies** [comment]
should we do a limited amount of people inside the game at a time? so we don't have like 20 people joining spleef at the same time?

Also an idea for this bug we could have a timer so there is a test for command and then if people join then after like 30 seconds it removes the button at the lobby, I think you can kind see where i'm going here

We can also make a game starting lobby in which once a certain amount of players join the game begins???sorta like hypixel????

**Comment by RedDragon98** [comment]
Yes, that is the eventual plan. We need to wait for gametest 1.1 to come out before we can make plugins like that I think
